doc/api-extensions.md

```diff
@@ -2344,3 +2344,7 @@ when working with volumes.
 ## `import_instance_devices`
 
 This API extension provides the ability to use flags `--device` when importing an instance to override instance's devices.
+## `instances_uefi_vars`
```

Comment:
doc-lint probably wants an empty line before this heading.

Reply:
Sure. Fixed!